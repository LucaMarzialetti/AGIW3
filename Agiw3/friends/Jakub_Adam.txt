--url_fb https://www.facebook.com/profile.php?id=100012884951657
--friends
VM6513:3 Aaron Driver
VM6513:3 Abdelkader Begoug
VM6513:3 Abdul Mannan
VM6513:3 Abi Majoka
VM6513:3 Abo Aline
VM6513:3 Abraham Lozoya
VM6513:3 Abs Hussain
VM6513:3 Achraf Alaoui Mzian
VM6513:3 Adam Castiel Burke
VM6513:3 Adam Lewis
VM6513:3 Adam Smith
VM6513:3 Ädïôs Ÿä Lćãvïä
VM6513:3 Adnan Çörten
VM6513:3 Adnan Fari
VM6513:3 Afzal Hossain
VM6513:3 Ahmad Al-said
VM6513:3 Ahmad Shezad
VM6513:3 Ahmed Alrrahal
VM6513:3 Ahmed Moula
VM6513:3 Ahmed Rashad
VM6513:3 Aimee Turoczy DS
VM6513:3 Aissa Rouissi
VM6513:3 Akande Micheal
VM6513:3 Alan Belmont
VM6513:3 Alana Dallas
VM6513:3 Alastair Francis Stewart
VM6513:3 Alex Featherstone
VM6513:3 Alex Tait
VM6513:3 Alexandre Santiago
VM6513:3 Alie B Bundu
VM6513:3 Alii Aras
VM6513:3 Allan John
VM6513:3 Amansingh Sangwan
VM6513:3 Ambria Calcavecchia
VM6513:3 Amelia Fatgirlslim Midgley
VM6513:3 Amin Mrabti
VM6513:3 Amine Amine
VM6513:3 Amine N Camara
VM6513:3 Amjad Abbas
VM6513:3 Andrew Beniston
VM6513:3 Andrew Burgess
VM6513:3 Andrew Hunt
VM6513:3 Andrew Price
VM6513:3 Andrew Priestley
VM6513:3 Andrew Roberts
VM6513:3 Andrew Shore
VM6513:3 Andrew Simond
VM6513:3 Andy Colley
VM6513:3 Andy Leather
VM6513:3 Andy Smith
VM6513:3 Anis Khlifi
VM6513:3 Anoop Patel
VM6513:3 Anthony Griff Griffiths
VM6513:3 Arkado Sj
VM6513:3 Arnold Essi
VM6513:3 Arturo Rodriguez
VM6513:3 Arvind Choudhary
VM6513:3 Asad Malik
VM6513:3 Asad Niazi
VM6513:3 Asemota Perry Martins
VM6513:3 Ash Danyal
VM6513:3 Ash Hill
VM6513:3 Ashely Cristine
VM6513:3 Ashfi Khan Hussain
VM6513:3 Avijeet Jayashekhar
VM6513:3 Ayoub Mekkaoui
VM6513:3 Aziz Mansuor
VM6513:3 Aziz Salawii
VM6513:3 Aziz Vich
VM6513:3 Azza Norton
VM6513:3 Azzo Zindaan Azzo
VM6513:3 Barrington Williams
VM6513:3 Barry Cummings
VM6513:3 Bash Iqbal
VM6513:3 Battas Mohamed
VM6513:3 Beaw Smilecute
VM6513:3 Ben Longley
VM6513:3 Ben Preece
VM6513:3 Bib Ahmed
VM6513:3 Billy Gibson
VM6513:3 Billy Tevnan
VM6513:3 Bob Bbm Smith
VM6513:3 Bobby Green
VM6513:3 Bobby Stewart
VM6513:3 Brahim Brahim
VM6513:3 Brahim Djani
VM6513:3 Brandon Orlando Willis
VM6513:3 Brendan Joyce
VM6513:3 Brian Langford
VM6513:3 Brian Richardson
VM6513:3 Brown S G Ruslar
VM6513:3 Bruce Gee
VM6513:3 Bruno Lourenço Fslb
VM6513:3 Callum Coltman
VM6513:3 Carl Schofield
VM6513:3 Carlo Moncada
VM6513:3 Carlos Piu
VM6513:3 Carter Sy
VM6513:3 Casey Holroyd
VM6513:3 Chadwick Harrison
VM6513:3 Charles Byrne
VM6513:3 Chemssou Chemchaoui
VM6513:3 Chris Anthony Benoit Adams
VM6513:3 Chris Brummitt
VM6513:3 Chris Bryan
VM6513:3 Chris Fear Ley
VM6513:3 Chris Holland
VM6513:3 Chris Manners
VM6513:3 Chris Souders
VM6513:3 Chris Sylvester
VM6513:3 Chris Wells
VM6513:3 Christina Rigden
VM6513:3 Christopher Bate
VM6513:3 Chuckd Heninger
VM6513:3 Cj Kc
VM6513:3 CJ Stokes
VM6513:3 Claudiu Isac
VM6513:3 Cliff Crosby
VM6513:3 Cliff Dent
VM6513:3 Clifford Morgan
VM6513:3 Cody Finnigan
VM6513:3 Colin Gordon
VM6513:3 Colin O'grady
VM6513:3 Craig Evans
VM6513:3 Craig Miller
VM6513:3 Craig Mycock
VM6513:3 Craig Rowbotham
VM6513:3 Cua Thoj
VM6513:3 D Ray Good
VM6513:3 Daksha Patel
VM6513:3 Dale Agg
VM6513:3 Dan Danny
VM6513:3 Dan Ken
VM6513:3 Dan Mustard
VM6513:3 Dan Spooner
VM6513:3 Dani Hasan
VM6513:3 Daniel Buckley
VM6513:3 Daniel Martin
VM6513:3 Darren Collins
VM6513:3 Darren Law
VM6513:3 Darren Lee Morris
VM6513:3 Darren Wiffa Smith
VM6513:3 Darryl Tasker
VM6513:3 Daryl Lee Towers
VM6513:3 David Adewale Adekanmbi
VM6513:3 David Burke
VM6513:3 David Donovan
VM6513:3 David Furd
VM6513:3 David Hetherington
VM6513:3 David Joel Carlson
VM6513:3 David Mack Rock
VM6513:3 David Mandela
VM6513:3 David Newton
VM6513:3 David Pettit
VM6513:3 David Stone
VM6513:3 David Willis
VM6513:3 Dean Deo Smu
VM6513:3 Dean Kemo Kempster
VM6513:3 Dean Martin
VM6513:3 Dean Mead
VM6513:3 Dean Stapleton
VM6513:3 Dean Utv Walton
VM6513:3 Dedy Ghepenk
VM6513:3 DèlLa Ljãñ
VM6513:3 Demetrius Goodie
VM6513:3 Dennis Okonta
VM6513:3 Derek Duval
VM6513:3 Derek Quinn
VM6513:3 Djawad Djawad
VM6513:3 Djilali Aïtali
VM6513:3 Djpux Morris
VM6513:3 Dominic Cranford
VM6513:3 Dominic Steel
VM6513:3 Donald Davidson
VM6513:3 Donald Farrington
VM6513:3 Donnie Mcdonald
VM6513:3 Duncan H Stanley Duncan
VM6513:3 Ebrima Kandeh
VM6513:3 Edu Blass Acuña
VM6513:3 Edwardo Gourzong
VM6513:3 Ehab S Mohamed
VM6513:3 El Mehdi Amara
VM6513:3 Emma Younique
VM6513:3 Emmanuel Osikorobia
VM6513:3 Enzo Alfieri
VM6513:3 Erick Nicolas
VM6513:3 Erkay Huseyin
VM6513:3 Eugene Bowmaster
VM6513:3 Eusebio Gomez
VM6513:3 Fairoz Miller
VM6513:3 Faisal Shaikh
VM6513:3 Fani Dunya
VM6513:3 Faraz Khan
VM6513:3 Fatah Ahmadi
VM6513:3 Fathi Ahmed
VM6513:3 Fatho Zr
VM6513:3 Firas Mskab Bawe
VM6513:3 Fleminem Flembo
VM6513:3 Francesco Cucullo
VM6513:3 Frankfelix Vasquez
VM6513:3 Fred Williams
VM6513:3 Gary Hunter
VM6513:3 Gary Jack
VM6513:3 Gary Mcgough
VM6513:3 Gary Scarborough
VM6513:3 Gav Whitaker
VM6513:3 Gavin Knowlton
VM6513:3 Gavin Webb
VM6513:3 Genti Peposhi
VM6513:3 George Macduff
VM6513:3 Ghano Etalon
VM6513:3 Gheyath Kraldy
VM6513:3 Giles Wilson
VM6513:3 Glady Win Gill
VM6513:3 Gold Diamond
VM6513:3 Graeme Innes
VM6513:3 Graham Miles
VM6513:3 Gregg Rogers
VM6513:3 Gregory Osborn Eromomene
VM6513:3 Guru Bishnoi
VM6513:3 Habz Hussian
VM6513:3 Hadii Moon
VM6513:3 Hakeem BossYazz Dibba
VM6513:3 Hamido Alazouzi
VM6513:3 Hamouda Spaie
VM6513:3 Harry Galdeira
VM6513:3 Harvey Dean
VM6513:3 Hassan Ben
VM6513:3 Hicham Rami
VM6513:3 Hoda Abohade
VM6513:3 Howard Davies
VM6513:3 Hubencio Del Rosario
VM6513:3 Hugo Silias
VM6513:3 Hukamran Faryadi
VM6513:3 Ian Flynn
VM6513:3 Ian Hilton
VM6513:3 İbrahim Fe
VM6513:3 Ibrar Anjum
VM6513:3 Icent Ogaga
VM6513:3 Ikechukwu Evans Ibeh
VM6513:3 Imran Hussain
VM6513:3 Iniesta Wahid Andres
VM6513:3 Irf Rasul
VM6513:3 Iskänder Jdäy
VM6513:3 Islam Sisan Al Harbi
VM6513:3 Jabbar Ahmed
VM6513:3 Jabed Hasan
VM6513:3 Jabeer El-fkirii
VM6513:3 Jack Evie May Collins
VM6513:3 Jack Patel
VM6513:3 Jack Reeder
VM6513:3 Jack Russell
VM6513:3 Jade Davies
VM6513:3 James Adam Fisher
VM6513:3 James Bacchiocchi
VM6513:3 James Belfitt
VM6513:3 James Carty
VM6513:3 James Kemsley
VM6513:3 James Marshall
VM6513:3 James Reason
VM6513:3 James Stanton
VM6513:3 James Timmins
VM6513:3 Jamie Boaden
VM6513:3 Jamie Halls
VM6513:3 Jason King
VM6513:3 Jason Vince
VM6513:3 Jason Wiggins
VM6513:3 Javed Mohd
VM6513:3 Jawad Sqawi
VM6513:3 Jay Ellahi
VM6513:3 Jay Exton
VM6513:3 Jed Mdaini
VM6513:3 Jeffrey Goodner
VM6513:3 Jenny Craig
VM6513:3 Jerome Rhodes
VM6513:3 Jesse Marshall
VM6513:3 JesseRey Martinez
VM6513:3 JH Valets
VM6513:3 Jim Douglas
VM6513:3 Jimmy Ariza
VM6513:3 Jo Simons
VM6513:3 Joe Da Lot
VM6513:3 Joe Moh
VM6513:3 Joe Paul Elkington
VM6513:3 John Adorkor
VM6513:3 John Blythe
VM6513:3 John Cizler
VM6513:3 John Galanopoulos
VM6513:3 John Greenaway
VM6513:3 John Hopkins
VM6513:3 John Jackson
VM6513:3 John Lee Baxter
VM6513:3 John Lucchesi
VM6513:3 John Mitchell Mitchell
VM6513:3 Johnsanderson Sanderson
VM6513:3 Johnson Kihal
VM6513:3 Jon Moon
VM6513:3 Jonathan Long
VM6513:3 Jonathan Rivero
VM6513:3 Jordan Lee
VM6513:3 Jordan Royston Jones
VM6513:3 Jose Lemus
VM6513:3 Jose Pina
VM6513:3 Jose Sebastian
VM6513:3 Josh Buffbutler
VM6513:3 Joundir Hasna
VM6513:3 Jousef Bakari
VM6513:3 Juan Garzia
VM6513:3 Juan Manuel Matias Mora
VM6513:3 Junaid Khan
VM6513:3 Kadi Bounab
VM6513:3 Kamal Singh
VM6513:3 Kamel Fares
VM6513:3 Kamel Mikhaiel
VM6513:3 Karen Allan
VM6513:3 Karl Holdsworth
VM6513:3 Kash Due
VM6513:3 Kayleigh Gardiner
VM6513:3 Keiran Ling
VM6513:3 Kelly Anderson
VM6513:3 Kelly Lewis
VM6513:3 Kenneth Hannah
VM6513:3 Kenneth Street Rider
VM6513:3 Kenny McKain
VM6513:3 Kevin Mccormick
VM6513:3 Kevin McGovern
VM6513:3 Kevin Vaughan
VM6513:3 Khan Khan Khan
VM6513:3 Khan Khan Mufeed
VM6513:3 Khizar KhiZrie
VM6513:3 Ki Ha
VM6513:3 Kieran James Thompson
VM6513:3 Kimberly Ann Martin
VM6513:3 Kirk Hodgkiss
VM6513:3 Kôptèr Dîdîn
VM6513:3 Kumar Mukesh
VM6513:3 Kyle Jones
VM6513:3 Lahcen Zalmati
VM6513:3 Lakhal Rachid
VM6513:3 Lamine Barry Barry
VM6513:3 Lance Footiefan
VM6513:3 Lb Hutchinson
VM6513:3 Lee Catterick
VM6513:3 Lee Catterick
VM6513:3 Lee Dancey
VM6513:3 Lee Green
VM6513:3 Lee Knight
VM6513:3 Lee Mears
VM6513:3 Lee Simpson
VM6513:3 Lee Watkins
VM6513:3 Leese Dooley
VM6513:3 Leobardo Zuna
VM6513:3 Les Underwood
VM6513:3 Liam Youson
VM6513:3 Lido Gara Amin
VM6513:3 Lizza Hills
VM6513:3 London Albania
VM6513:3 Lory Hernandez
VM6513:3 Lotfi Riahi De Zaghouan
VM6513:3 Lovemohd Mohd
VM6513:3 Luis Salas
VM6513:3 Luthfur Rahman
VM6513:3 M George Brace
VM6513:3 Magil Brazil
VM6513:3 Malaak El Hoop
VM6513:3 Malik Fsisal
VM6513:3 Malik Raswl
VM6513:3 Mandm Mirrors
VM6513:3 Mark AJ Terry
VM6513:3 Mark Anthony
VM6513:3 Mark Johnson
VM6513:3 Mark Jones
VM6513:3 Mark Lacey
VM6513:3 Mark Matthews
VM6513:3 Mark Murphy
VM6513:3 Mark Prevatt
VM6513:3 Marley C Knox
VM6513:3 Marley MB Bob
VM6513:3 Martin Cadogan
VM6513:3 Matt Daniels
VM6513:3 Matthew Goode
VM6513:3 Matthew Green
VM6513:3 Matthew Pointon
VM6513:3 Matthew Wallinger
VM6513:3 Mc Willz E B
VM6513:3 Med Bouaraba
VM6513:3 Med Monica Christensen
VM6513:3 Mehdi Lahab
VM6513:3 Meshu Angle
VM6513:3 Mhamad Kwestani
VM6513:3 Mīă Cølpå
VM6513:3 Michael Button
VM6513:3 Michael Comerford
VM6513:3 Michael Heywood
VM6513:3 Michael Jones
VM6513:3 Michael Price
VM6513:3 Michael Rowe
VM6513:3 Michael Whitehorn
VM6513:3 Mickey Soula
VM6513:3 Mico Mitchell
VM6513:3 Midou Salem
VM6513:3 Miguel Cruz Gustos
VM6513:3 Mike Healey
VM6513:3 Mike Murray
VM6513:3 Mike Yahya
VM6513:3 Mikee Canny
VM6513:3 MJ Raffles
VM6513:3 Mohamed Ali
VM6513:3 Mohamed Bachoua
VM6513:3 Mohamed Ingrachen
VM6513:3 Mohamed Zannouti
VM6513:3 Mohammed Bougeunna
VM6513:3 Mohammed Parvez
VM6513:3 Mohandakli Saadi
VM6513:3 Mohd Roshan
VM6513:3 Monjur Ahmed Mamun
VM6513:3 Monsef Elsalhi
VM6513:3 Moss Ramzy
VM6513:3 Mouhamad Lkhadar
VM6513:3 Mouloud Kermiche
VM6513:3 Mozammel Hossain
VM6513:3 Mukesh Sahu
VM6513:3 Mun E Mike
VM6513:3 Munna Patel
VM6513:3 Mv Gh
VM6513:3 Nad Akhtar
VM6513:3 Nadia Umer
VM6513:3 Natalie Robinson
VM6513:3 Natasha Gremore
VM6513:3 Nathan Paxman
VM6513:3 Naveed Ahmad
VM6513:3 Neill Tucker
VM6513:3 Nelson A Daria
VM6513:3 Nev West
VM6513:3 Nick Beavan
VM6513:3 Nick Gull
VM6513:3 Nick Marshall
VM6513:3 Nicu Cretu
VM6513:3 Nigel Booth
VM6513:3 Nina Anderson
VM6513:3 Nizar El
VM6513:3 Noel Lohia
VM6513:3 Noor Johnny
VM6513:3 Norman Caines
VM6513:3 Norman Paul
VM6513:3 Nouayem Chaouch
VM6513:3 Nour Othman
VM6513:3 Nusrullah Balouch
VM6513:3 Nweke Kelechi
VM6513:3 Obidike Paulinus
VM6513:3 Obsolet Jeff Hardy
VM6513:3 One'slife Compainon
VM6513:3 Ony Sllu Chyg Sani
VM6513:3 Osaberimah Nana Qwequ SarkCess
VM6513:3 Parfait Alfrez Ekoue
VM6513:3 Patrick Curley
VM6513:3 Patrick King
VM6513:3 Patrick O'Rourke
VM6513:3 Paul Adams
VM6513:3 Paul Brown
VM6513:3 Paul Childs
VM6513:3 Paul David Wood
VM6513:3 Paul Fuller
VM6513:3 Paul Kirk
VM6513:3 Paul Pirlou
VM6513:3 Paul Ritchie
VM6513:3 Paul Thompson
VM6513:3 Paul Tirella
VM6513:3 Paul Tommis
VM6513:3 Paul Williams
VM6513:3 Paul Wilmot
VM6513:3 Pederson Langford
VM6513:3 Pedro Lozada
VM6513:3 Peshrow Kured
VM6513:3 Peter Alm
VM6513:3 Peter Cee
VM6513:3 Peter Greenwood
VM6513:3 Peter Scally
VM6513:3 Phillip Phillip
VM6513:3 Prashant Mishra
VM6513:3 Preet Harman
VM6513:3 Queen-Ti Chester
VM6513:3 Radoslav Biserov
VM6513:3 Raf Modric
VM6513:3 Rafiq Rafiq
VM6513:3 Rahul Sharma
VM6513:3 Ralph Hplar
VM6513:3 Ratnesh Tandel
VM6513:3 Ray Obrien
VM6513:3 Raymond Abela
VM6513:3 Raza Miah
VM6513:3 Reco Holders
VM6513:3 Red Spook
VM6513:3 René Loick
VM6513:3 Ricardo Viveiros
VM6513:3 Richard Burch
VM6513:3 Richard Croudace
VM6513:3 Richard Pip Vincent
VM6513:3 Richard Pugh
VM6513:3 Richard Roberts