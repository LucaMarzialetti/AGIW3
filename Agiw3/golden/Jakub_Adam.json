{
    "PATTERN": {
        "tel": [
            "882 070 414",
            "5732714316",
            "243410043"
        ],
        "addr": [],
        "email": ["trenerjakubadam@gmail.com"]
    },
    "NER": {
        "LOC": ["Obozy Oferta zamowienie"],
        "ORG": [],
        "PER": [
            "Analiza Dodatki",
            "Jakub Adam",
            "Komunikacja"
        ]
    },
    "url": "http://jakubadam.pl/oferta/"
}